### PR TITLE
Create display comments popup

### PR DESCRIPTION
**Pull request by Lembani**
Highlights: 
- Add popup modal markup and functionality
- Add modal styles
- etc

Your review will highly be appreciated. 🙏🏾

### CODE REVIEW

**Review by GedeonTS — APPROVED** [review]
# Approved ✅ 🥳 🎉 

Hi @Lembani , 👋🏻 

Your project is complete!  
There is nothing else to say other than... it's time to merge it  🚀 

Well done! Congratulations! 🎉 



# (Highlights) Good points

- Good use of ES6 syntax.
- No linter errors.
- The PR is following the Github workflow.
- Good commit messages.

## Optional Suggestions

- Kindly resize the image in the popup, because this would make your project look good.
 - It would be better if you added comments in your code because it would make it easier to understand for others.
 
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding! 👏👏👏

**Comment by Lembani** [comment]
Thanks a lot for the feedback. 👍🏾 
I appreciate and I will make the suggested improvements.

File: src/modules/middleSection.js [hunk 1]
```diff
@@ -7,14 +8,68 @@ const fetchAll = async () => {
...
+  commentBtns.forEach((commentBtn) => {
+    commentBtn.addEventListener('click', () => {
+      const btnId = commentBtn.getAttribute('id');
+      const movieDetails = arr[btnId];
+
+      const modal = `
+        <div class="header-container">
+          <div class="img-container">
+            <img src="${movieDetails.image.original}" class="mov-img" alt="Movie Image">
+          </div>
```
Comment (GedeonTS) [inline src/modules/middleSection.js:33]: [OPTIONAL] I would suggest to resize the image to make it fit the container.

Comment (Lembani) [reply src/modules/middleSection.js:33]: Noted thanks, will do as suggested.

**Pull request merged**